tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -626,9 +626,9 @@ describe( 'Shopper → Mini Cart', () => {
 				);
 
 				await expect( page ).toMatchElement(
-					'.wc-block-mini-cart__title',
+					'.wc-block-mini-cart__footer-cart',
 					{
-						text: 'Je winkelwagen (1 stuk)',
+						text: 'Bekijk mijn winkelwagen',
```

Comment:
Due to an [issue with `wp i18n`](https://github.com/wp-cli/i18n-command/issues/324), plural terms aren't copied when updating a string. We switched to checking for non-plural terms till that issue is fixed.